hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/services/attributesearch/AttributeSearchIntegrationTest.kt

```diff
@@ -267,7 +267,7 @@ class AttributeSearchIntegrationTest : AbstractSearchIntegrationTest() {
         }
       }
 
-      webTestClient.attributeSearch(request).expectPrisoners("P1", "P2", "P3")
+      webTestClient.attributeSearch(request).expectPrisoners("P1")
```

Comment:
A few of the tests based on names have different expectations due to the name changes.